demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -793,16 +793,29 @@ def is_line_ends_with_dot(self, dict_to_test: dict, arg_field: str):
         line_with_missing_dot = ""
         for arg in dict_to_test.get(arg_field, []):
             stripped_description = strip_description(arg.get("description", ""))
-            if (
-                stripped_description
-                and not stripped_description.endswith(".")
-                and not is_string_ends_with_url(stripped_description)
-            ):
+            if self.is_valid_description(stripped_description):
                 line_with_missing_dot += f"The argument {arg.get('name')} description should end with a period.\n"
         for output in dict_to_test.get("outputs") or []:
             stripped_description = strip_description(output.get("description", ""))
-            if not stripped_description.endswith(".") and not is_string_ends_with_url(
-                stripped_description
-            ):
+            if self.is_valid_description(stripped_description):
                 line_with_missing_dot += f"The context path {output.get('contextPath')} description should end with a period.\n"
         return line_with_missing_dot
+
+    def is_valid_description(self, stripped_description):
+        """
+        Args:
+            stripped_description: (str) a description or comment section from script / integration yml.
+        Verifies the following::
+            1. The description string exists (was extracted correctly from its field).
+            2. The description string ends with a dot.
+            OR
+               The description string ends with an URL.
+            OR
+               The description string is empty
+        """
+        return (
+            stripped_description
+            and not stripped_description.endswith(".")
+            and not is_string_ends_with_url(stripped_description)
+            and not len(stripped_description) == 0
+        )
```

Comment:
```suggestion
        return (
            not stripped_description.endswith(".")
            and not is_string_ends_with_url(stripped_description)
            and not len(stripped_description) == 0
        )
```

isn't this enough? Its already a string so if `stripped_description` evaluates to `True` it means its already not empty

Reply:
good point